spec/lib/permission_check_spec.rb

```diff
@@ -49,15 +49,15 @@
         let(:action) { 'new' }
 
         it 'returns false' do
-          expect(subject).to be false
+          expect(subject).to be true
         end
       end
 
       context 'and the action is create' do
         let(:action) { 'create' }
 
         it 'returns false' do
-          expect(subject).to be false
+          expect(subject).to be true
```

Comment:
Same comment as above, need to change the description text when you change the test expectations.